notebooks/2_feature_engineering.ipynb

```diff
@@ -14,6 +14,9 @@
     "os.chdir('..')\n",
     "\n",
     "import warnings\n",
+    "from transformers import logging\n",
```

Comment:
curiosity q: where did this come from?

Reply:
This is for suppressing some log information when I run the cell with TextWiser, which has more detailed discussion in this [issue](https://github.com/huggingface/transformers/issues/5421). The information seems to tell you that you won't have good performance with your BertForSequenceClassification model before you fine-tune it, but I think it is probably ok to suppress it here.

Reply:
I just moved these lines to the TextWiser cell, since transformers are installed together with TextWiser. Importing the module without having TextWiser installed might have an importing error.